tests/e2e/utils/payments.js

```diff
@@ -56,19 +56,24 @@ export async function fillCardDetails( page, card ) {
 			{ timeout: 30000 }
 		);
 		await cardNumberInput.type( card.number, { delay: 20 } );
+		await page.waitFor( 1000 );
```

Comment:
This test would sometimes fail (especially in headless mode, rarely in interactive mode) due to Puppeteer not typing the full value in the field. It would be missing a few numbers.
I find that adding these pauses helped with the stability of this test overall, although not sure why it's needed in the first place